docs/source/faq.rst

```diff
@@ -0,0 +1,114 @@
+==========================
+Frequently Asked Questions
+==========================
+
+This page contains frequently asked questions about MOTECH with short answers to them.
+
+.. contents::
+    :depth: 2
+
+Can Motech run on Windows?
+-----------------------------
```

Comment:
The length of underline must exactly match the number of characters in the header, to avoid warnings